langchain4j-core/src/main/java/dev/langchain4j/rag/DefaultRetrievalAugmentor.java

```diff
@@ -264,12 +272,16 @@ private static void log(Query query, ContentRetriever retriever, List<Content> c
         log.debug("Retrieved {} contents using query '{}' and retriever '{}'",
                 contents.size(), query.text(), retriever);
 
-        if (contents.size() > 0) {
+        if (!contents.isEmpty()) {
```

Comment:
can we add here ` && log.isTraceEnabled()` to not build contentsString when logging was not requested?